styles/main.css

```
@@ -0,0 +1,244 @@
+.movie-search > input {
+  background-color: transparent;
+  border: none;
+  width: 100%;
+  color: var(--color-white);
+  outline: none;
+}
+
+.movie-search > input::placeholder {
+  color: var(--color-white);
+
+  font-size: 1rem;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)